modules/currentTime.js

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from './luxon.js';
+
+function currentTime() {
+  const timeEl = document.querySelector('time');
+
+  setInterval(() => {
+    const now = DateTime.now();
+    timeEl.innerText = now.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  }, 1000);
+}
```

Comment:
- [ ] Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
Done. Thanks